Tasks/DownloadTeamCityArtifacts/task.json

```diff
@@ -0,0 +1,112 @@
+{
```

Comment:
This task should also live in vsts-rm-extensions & get bundled with "TeamCity artifacts" extension